LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2EventListener_XP.uc

```diff
@@ -19,6 +19,8 @@ var config float TOP_RANK_XP_TRANSFER_FRACTION;
 
 var config int LISTENER_PRIORITY;
 
+var config array<name> INELIGIBLE_FOR_MISSION_EXP;
```

Comment:
Please use `XP` or `Xp` for the short form of "experience", for consistency with most(?) other code:
```suggestion
var config array<name> INELIGIBLE_FOR_MISSION_XP;
```

Reply:
Actually, for clarity I suggest including "classes" in the name. Otherwise it's not clear what the config setting is for. For example, `CLASSES_INELIGIBLE_FOR_MISSION_XP` is clearer, if longer.